bitnami/kuberay/README.md

```diff
@@ -0,0 +1,562 @@
+<!--- app-name: KubeRay -->
+
+# KubeRay packaged by Bitnami
+
+KubeRay is a Kubernetes operator for deploying and management of Ray applications on Kubernetes using CustomResourceDefinitions.
+
+[Overview of KubeRay](https://www.ray.io/)
+
+Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
+
+## TL;DR
```

Comment:
It seems this README is not using the latest changes implemented by @jotamartos at https://github.com/bitnami/charts/pull/21009. From that moment, we should follow that structure for all existing and new Helm charts

Reply:
Fixed